vue-frontend/src/components/jobs/FaqSection.vue

```diff
@@ -348,10 +351,11 @@ button[disabled] {
 @include media-breakpoint-up(md) {
   .row {
     margin: 0 6%;
+    margin-left: -2rem;
   }
 
   .faq-content {
-    padding-left: 2rem;
+    padding-left: 5rem;
```

Comment:
Any reason we are using padding in rem? I thought we will use it only for font size?